job-service/src/main/java/com/hpe/caf/services/job/api/ApiExceptionMapper.java

```diff
@@ -40,14 +47,22 @@
      */
     @Override
     public Response toResponse(Exception exception) {
+        LOG.debug("exception found {}", exception.getClass());
```

Comment:
Maybe get rid of this debug log, I assume it was just for testing purposes?

Reply:
Removed